src/App.js

```diff
@@ -3,6 +3,7 @@ import React, { Suspense } from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import Navbar from './Components/Navbar/Navbar';
 import CitiesPage from './Pages/CitiesPage/CitiesPage';
+import CITIES from './MockData/cities.json';
```

Comment:
this is `cities` not `CITIES`
```suggestion
import cities from './MockData/cities.json';
```